package.json

```
@@ -72,9 +72,8 @@
   },
   "lint-staged": {
     "*.ts": [
-      "prettier --config ./node_modules/resin-lint/config/.prettierrc --write",
-      "resin-lint --typescript",
-      "git add"
+      "balena-lint --typescript --fix",
+      "balena-lint --typescript"
```

The `--fix` also does a check at the same time:
```suggestion
      "balena-lint --typescript --fix"
```